cves/CVE-2017-9787.yml

```diff
@@ -45,7 +45,8 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: A specific type of proxy could be used to make many connections and
```

Comment:
Very vague. Could elaborate a lot more into the types of proxy. Number of connections. What causes the system to go down.